### ISSUE TITLE
Dataset at ftp:/ftp.cpc.ncep.noaa.gov/precip/global_CMORPH

### ISSUE THREAD

**Issue by nickrsan** [climate-mirror/datasets#161]
ftp:/ftp.cpc.ncep.noaa.gov/precip/global_CMORPH.

Suggested in a large email containing many urls

**Comment by claughinghouse** [comment]
Downloading.

**Comment by donbright** [comment]
got this message when trying to connect
```
don@miriam:/var/www/html$ ftp ftp.cpc.ncep.noaa.gov
Connected to ftp-cpc.ncep.noaa.gov.
220-******************************************************************
220-                 **WARNING**WARNING**WARNING**
220-This is a United States (Agency) computer system, which may be accessed
220-and used only for official Government business by authorized personnel.
220-Unauthorized access or use of this computer system may subject violators
220-to criminal, civil, and/or administrative action.

```

standard anonymous login gives this

ftp> ls
550 Permission denied.
ftp: bind: Address already in use

**Comment by empirical-bayesian** [comment]
That's boilerplate, and invalid if interpreted that way. It is a public
system. What's wrong is the posting of the boilerplate.


On Sun, Jan 29, 2017, at 21:59, don bright wrote:

> got this message when trying to connect


> don@miriam:/var/www/html$ ftp ftp.cpc.ncep.noaa.gov Connected to ftp-
> cpc.ncep.noaa.gov. 220-
> ****************************************************************** 220-
> **WARNING**WARNING**WARNING** 220-This is a United States (Agency)
> computer system, which may be accessed 220-and used only for official
> Government business by authorized personnel. 220-Unauthorized access
> or use of this computer system may subject violators 220-to criminal,
> civil, and/or administrative action.

>
> standard anonymous login gives this



> ftp> ls

>  550 Permission denied.

>  ftp: bind: Address already in use



> — You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub[1], or mute the
> thread[2].
> 




Links:

  1. https://github.com/climate-mirror/datasets/issues/161#issuecomment-275970876

**Comment by donbright** [comment]
but are there any files to get? the system seems unresponsive...

**Comment by bkirkbri** [comment]
@donbright Looks like it available now:

```
lftp ftp.cpc.ncep.noaa.gov:/precip/global_CMORPH> du -h .
2.1G    ./3-hourly_025deg
4.0G    ./30min_025deg
16G     ./30min_8km
0       ./cmorph_temp
945M    ./daily_025deg
0       ./recovery/30min_8km
42M     ./recovery/daily_025deg
42M     ./recovery
23G     .
```

**Comment by bkirkbri** [comment]
Downloading now...

**Comment by wantonwonton** [comment]
I've just started a download as well (to be an offline copy).

**Comment by bkirkbri** [comment]
@wantonwonton Did you get a copy?

**Comment by wantonwonton** [comment]
Yes, my download finished.  I have a hashdeep file (about 2MB compressed), but I don't have a convenient place to put it (unless I upload it as an attachment here).  Any recommendations?

**Comment by donbright** [comment]
if you have a public ssh key i can give you sftp access to my VM 67.205.151.30

**Comment by wantonwonton** [comment]
@donbright Thanks for the suggestions.

You can find my hashdeep results (compressed with bzip2) [here](https://filebin.net/75r414twt8tiswjo).

**Comment by bkirkbri** [comment]
Completed and available at:

Index: 
https://s3.ca-central-1.amazonaws.com/climate-mirror/ftp.cpc.ncep.noaa.gov/ftp.cpc.ncep.noaa.gov_precip_global_CMORPH.index.html

Hashdeep: 
https://s3.ca-central-1.amazonaws.com/climate-mirror/ftp.cpc.ncep.noaa.gov/ftp.cpc.ncep.noaa.gov_precip_global_CMORPH.hashdeep.txt

Hashdeep signature: 
https://s3.ca-central-1.amazonaws.com/climate-mirror/ftp.cpc.ncep.noaa.gov/ftp.cpc.ncep.noaa.gov_precip_global_CMORPH.hashdeep.txt.asc

**Comment by bkirkbri** [comment]
@wantonwonton @donbright Thanks! I'll compare hashes tomorrow.
